src/actions.js

```diff
@@ -341,7 +376,9 @@ export function getActionDefinitions(self) {
 			name: 'Lens - Follow Focus',
 			options: optSetIncDecStep('Focus setting', 0x555, 0x0, 0xaaa, 10),
 			callback: async (action) => {
-				await self.getPTZ('AXF' + cmdValue(action, 0x555, 0x0, 0xaaa, action.options.step, 3, self.data.focusPosition))
+				const parsedAction = await parseSetIncDecVariables(action, self, 0x0, 0xaaa, 10)
+				if (parsedAction === undefined) return
```

Comment:
Given your refactoring of `parseIncDecVariables`, and conforming NaN values to min, `parsedAction` should no longer ever resolve to undefined and these checks can be done away with.

I typically abort an action where a totally invalid variable is supplied, hence the checks..

Reply:
You are right.
Hopefully fixed it again now.